Arrowgene.Ddon.GameServer/Handler/ClanClanGetJoinRequestedListHandler.cs

```diff
@@ -1,25 +1,20 @@
-﻿using Arrowgene.Ddon.GameServer.Dump;
 using Arrowgene.Ddon.Server;
-using Arrowgene.Ddon.Server.Network;
-using Arrowgene.Ddon.Shared.Network;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
 using Arrowgene.Logging;
 
 namespace Arrowgene.Ddon.GameServer.Handler
 {
-    public class ClanClanGetJoinRequestedListHandler : PacketHandler<GameClient>
+    public class ClanClanGetJoinRequestedListHandler : GameRequestPacketHandler<C2SClanClanGetJoinRequestedListReq, S2CClanClanGetJoinRequestedListRes>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(ClanClanGetJoinRequestedListHandler));
 
-
         public ClanClanGetJoinRequestedListHandler(DdonGameServer server) : base(server)
         {
         }
 
-        public override PacketId Id => PacketId.C2S_CLAN_CLAN_GET_JOIN_REQUESTED_LIST_REQ;
-
-        public override void Handle(GameClient client, IPacket packet)
+        public override S2CClanClanGetJoinRequestedListRes Handle(GameClient client, C2SClanClanGetJoinRequestedListReq request)
         {
-            client.Send(InGameDump.Dump_69);
```

Comment:
Should we keep the pcap here in comments if it has some details we might need in the future? (to remember this is related to this packet)